src/components/CategoryGameList.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from "react";
+import { useContext } from "react";
+import { getCategoryReviews } from "../functions/api";
+import Categories from "./Categories";
+import { useParams } from "react-router-dom";
+
+const CategoryGameList = () => {
+  const { category } = useParams();
+  const [gamesList, setgamesList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  useEffect(() => {
+    setIsLoading(true);
+    getCategoryReviews(category).then((res) => {
+      setgamesList(res);
+      setIsLoading(false);
+    });
+  }, []);
+
+  return (
+    <div>
+      <Categories />
+      {isLoading ? (
+        <p>Loading</p>
+      ) : (
+        <div className="GameList">
+          {gamesList.map((review) => {
+            return (
+              <div className="grid-review" key={review.review_id}>
+                <img className="images" src={review.review_img_url} />
+                <li>
+                  <ul>
+                    <b>{review.title}</b>
+                  </ul>
+                  <ul>Category: {review.category} </ul>
+                  <ul>Designer: {review.designer} </ul>
+                  <ul>Review author: {review.owner}</ul>
+                </li>
+                <li>
+                  <ul>Votes: {review.votes}</ul>
+                  <ul>Comments: {review.comment_count}</ul>
+                </li>
```

Comment:
`<li>` tags should be in a `<ul>` or an `<ol>` tag not the other way around!

It would probably be best to make the `<div className="GameList">` into your list component (`ol` or `ul`) and have the first child inside the map an `<li>`


```diff
-       <div className="GameList">
+       <ul className="GameList">
          {gamesList.map((review) => {
            return (
-              <div className="grid-review" key={review.review_id}>
+              <li className="grid-review" key={review.review_id}>
```